docs/source/config/ui.rst

```diff
@@ -120,8 +123,12 @@ The section has the following syntax:
         - id: video_size
           show: <show>
 
-It is a list of all supported sample video widgets.
-``<show>`` flag is a boolean, when it's *true* the widget is included, when *false* it is omitted.
+It consists of two keys:
+- `grid_settings` - settings related to the mesh grid controls in the `draw_grid` component.
+- `components` - a list of all supported sample video widgets, in the format:
+  - id: widget id
+  - show: <show> - boolean flag, when it's *true* the widget is included, when *false* it is omitted.
+
```

Comment:
The rendered document look a bit odd:

![image](https://github.com/user-attachments/assets/42fa52f2-4c0a-4364-bcfe-1003538051e7)

You can see the result here: https://mxcubeweb--1458.org.readthedocs.build/en/1458/config/ui.html#sample-view-video-controls